pyVor/tests/testPrimitives.py

```diff
@@ -271,6 +271,11 @@ def test_to_vector(self):
         self.assertEqual(self.b.to_vector(), Vector(1, 2))
         self.assertNotEqual(self.b.to_vector(), Vector(1, 2, 0))
 
+    def test_to_array(self):
```

Comment:
could you delete it? cause it seems pretty silly to test that a function doesn't exist